packages/perseus-editor/src/widgets/image-editor.tsx

```diff
@@ -329,7 +329,11 @@ const ImageEditor: any = createReactClass({
             this.setUrl(url, size[0], size[1], true);
         } catch (error) {
             this.setState({
-                backgroundImageError: `There was an error loading the image URL: ${error}`,
+                backgroundImageError: `There was an error loading the image URL ${JSON.stringify(
```

Comment:
I think the colon is still useful here. :) 

```suggestion
                backgroundImageError: `There was an error loading the image URL: ${JSON.stringify(
```